spec/quality_spec.rb

```diff
@@ -115,19 +131,29 @@ def check_for_specific_pronouns(filename)
   end
 
   it "does not include any leftover debugging or development mechanisms" do
-    included = /spec/
-    exempt = /quality\_spec\.rb/
+    exempt = %r{quality_spec.rb|support/helpers}
```

Comment:
If you're referring to the reflex change to `%r`, it was to appease Rubocop. Not 100% sure why usage of `|` triggers Rubocop to suggest using `%r`, though I suspect it's actually the usage of `/` that triggers it.